cypress/e2e/item/view/viewFolder.cy.ts

```diff
@@ -89,11 +89,25 @@ describe('View Folder', () => {
 
   it('search', () => {
     const { id } = parentItem;
+    const searchText = child1.name;
     cy.visit(buildItemPath(id, { mode: ItemLayoutMode.Grid }));
 
     cy.get(`#${buildItemCard(child1.id)}`).should('be.visible');
 
-    cy.get(`#${ITEM_SEARCH_INPUT_ID}`).type(child1.name);
+    cy.wait('@getChildren');
+    cy.wait('@getChildren');
```

Comment:
Why do you need these two here ? Maybe add a comment if you really need to wait on them.
Also, I think the following should work too.

```suggestion
    cy.wait(['@getChildren', '@getChildren']);
```